Documentation/specs/runtime-configuration-file.md

```diff
@@ -118,7 +119,10 @@ This section is copied verbatim from an identical section in the input `project.
 
 * `gcServer` - Boolean indicating if the server GC should be used (Default: _TBD_). Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/ms229357.aspx) setting)
 * `gcConcurrent` - Boolean indicating if background garbage collection should be used (Default: _TBD_). Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/yhwwzef8.aspx) setting).
-* `framework` - Indicates the name and version of the shared framework to use when activating the application. The presence of this section indicates that the application is a portable app designed to use a shared redistributable framework.
+* `framework` - Indicates the `name`, `version` and other properties of the shared framework to use when activating the application. The presence of this section indicates that the application is a portable app designed to use a shared redistributable framework.
```

Comment:
Suggestion: Don't imply the presence of the section for standalone apps. You might want to add to the end ...
```
This section will not be present for standalone applications that do not rely upon a shared framework. 
```
Nit: Lists have (for the most part) adopted the comma prior to the conjunction format:
```
the `name`, `version`, and other properties
```